modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/related/models/test/ObjectRelatedModelsProviderTest.java

```diff
@@ -1136,6 +996,173 @@ private void _testOrganizationSystemObjectEntry1toMObjectRelatedModelsProviderIm
 		_organizationLocalService.deleteOrganization(organization1);
 	}
 
+	private void _testOrganizationSystemObjectEntryMtoMObjectRelatedModelsProviderImpl()
```

Comment:
Again, this method has a lot of duplicate code with `_testUserSystemObjectEntryMtoMObjectRelatedModelsProviderImpl()`, can we make this method more generic and use it for both? this way we can reuse a lot of code.